pyproject.toml

```diff
@@ -58,6 +58,11 @@ pytest-pylint = "^0.17.0"
 requests_mock = "^1.8.0"
 tox = "^3.14.5"
 gitlint = "^0.15.1"
+types-freezegun = "^0.1.4"
+types-pytz = "^2021.1.0"
+types-requests = "^2.25.0"
+types-six = "^0.1.7"
+mypy = "v0.902"
```

Comment:
`v0.910` is latest.

Reply:
I wasn't able to use v0.910 due to the wheels archive being corrupt.
```
mypy-0.910-cp38-cp38-manylinux2010_x86_64.whl'] errored with the following return code 2, and output: 
  Processing /home/user/.cache/pypoetry/artifacts/0c/4d/47/70f60992a00848d064dc46fed832b489481066bb9149db21ab8b910b33/mypy-0.910-cp38-cp38-manylinux2010_x86_64.whl
  ERROR: Exception:
  Traceback (most recent call last):
...
    File "/usr/lib/python3.8/zipfile.py", line 1336, in _RealGetContents
      raise BadZipFile("File is not a zip file")
  zipfile.BadZipFile: File is not a zip file
```